src/components/primitive/chip-input/chip-input.component.tsx

```diff
@@ -1,92 +1,86 @@
 import { useState } from 'react';
 import styled from 'styled-components';
+import { t } from 'i18n-js';
+import { Input } from '../input/input.component';
+import { Box } from '../box/box.component';
 import Chip from '../chip/chip.component';
 import { ChipInputComponentProps } from './chip-input.component.props';
 
-const ChipContainer = styled.div`
-  margin-top: 1rem;
-  display: grid;
+const ChipContainer = styled(Box)`
+  margin-top: 1rem !important;
+  display: grid !important;
   grid-template: repeat(6, 1fr) / repeat(6, 1fr);
 
   @media screen and (max-width: 768px) {
-    display: flex;
+    display: flex !important;
     flex-direction: column;
   }
 `;
 
-const Container = styled.div<{ length: number }>`
+const Container = styled(Box)<{ length: number }>`
   margin: 1rem !important;
   border: 2px solid #cbccdc;
   border-radius: 5px;
   width: 100%;
-  padding: 1rem !important;
+  padding: 0 1rem !important;
   background-color: #fff;
   height: ${({ length }) =>
-    length === 0 ? '5rem' : length > 6 ? '16rem' : '11rem'};
+    length === 0 ? '7rem' : length > 6 ? '18rem' : '13rem'} !important;
 
   @media screen and (max-width: 768px) {
     height: ${({ length }) =>
-      length === 0 ? '5rem' : `${length * 45 + 70}px`};
+      length === 0 ? '7rem' : `${length * 5 + 11}rem`} !important;
```

Comment:
The calculations above are unclear. Either use appropriate constants, custom function, or add code comments.